src/styles/components/_home-search.scss

```diff
@@ -0,0 +1,87 @@
+@import '../abstracts/_abstracts-dir';
+
+.home-search {
+  display: flex;
+  justify-content: flex-end;
+  height: rem(250px);
+
+  @include respond-above('sm') {
+    height: rem(375px);
+  }
+
+  @include respond-above('md') {
+    height: rem(500px);
+  }
+  
+  @include respond-above('lg') {
+    height: rem(600px);
+  }
+
+  @include respond-above('xl') {
+    height: rem(700px);
+  }
+
+  &__content {
+    display: flex;
+    justify-content: space-around;
+    flex-direction: column;
+    max-width: 47.5%;
+  
+    @include respond-above('sm') {
+      max-width: 50%;
+      margin-bottom: 0;
+    }
+  
+    @include respond-above('lg') {
+      justify-content: space-between;
+      max-width: 55%;
+    }
+  }
+
+  &__title {
+    margin: 0;
+    font-size: rem(24px);
+
+    @include respond-above('md') {
+      font-size: rem(45px);
+    }
+
+    @include respond-above('lg') {
+      font-size: rem(60px);
+    }
+
+    @include respond-above('xl') {
+      font-size: rem(75px);
+    }
+  }
+
+  &__description {
+    padding-bottom: 0;
```

Comment:
We don't need to override this since it does not have the [default value](https://www.w3schools.com/cssref/css_default_values.asp)